### DISCUSSION TITLE
Don't never buy no weed from the gas station

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#431] [General]
Little oopsie I committed in the charter
![image](https://github.com/user-attachments/assets/b5d91edc-a9ca-41a5-a9c9-1dda96405c78)

**Comment by r6915ee** [comment]
My only question is:

what

**Reply by THEkatinamicrowave** [reply]
haxeflixel debug console shenanigans

**Reply by r6915ee** [reply]
truly a rabbit hole of a debugger
